ui/Cargo.toml

```diff
@@ -20,6 +20,8 @@ warp = { git = "https://github.com/Satellite-im/Warp", rev = "ed3b5de968fb623a7d
 warp-mp-ipfs = { git = "https://github.com/Satellite-im/Warp", rev = "ed3b5de968fb623a7d76170a5d7dd3f3c79d3e1b" }
 warp-rg-ipfs = { git = "https://github.com/Satellite-im/Warp", rev = "ed3b5de968fb623a7d76170a5d7dd3f3c79d3e1b" }
 warp-fs-ipfs = { git = "https://github.com/Satellite-im/Warp", rev = "ed3b5de968fb623a7d76170a5d7dd3f3c79d3e1b" }
+rfd = {version = "0.10.0", default-features = false, features = ["xdg-portal"] }
```

Comment:
I wonder if the feature was the reasoning dialogs was no longer working for me. If i recall, rfd default feature is using gtk. Just curious, but any reason to use `xdg-portal` over what was default?